### Segmenting and Clustering Neighborhoods in Toronto - Part 2

#### Using csv data in http link for geospatial data

In [4]:
import pandas as pd # library for data analsysis
import numpy as np # library to handle data in a vectorized manner

link = "http://cocl.us/Geospatial_data"
df_new = pd.read_csv(link)

df_new.head()

Postal Code   Latitude  Longitude
0         M1B  43.806686 -79.194353
1         M1C  43.784535 -79.160497
2         M1E  43.763573 -79.188711
3         M1G  43.770992 -79.216917
4         M1H  43.773136 -79.239476

In [5]:
df_new.shape

(103, 3)

Both the DFs have 103 rows and 3 columns.


#### Merging postal code and post code 

In [6]:
df_new.columns = ['Postcode','Latitude','Longitude']

cols = df_new.columns.tolist()
cols

['Postcode', 'Latitude', 'Longitude']

#### getting df  from previous 

In [7]:
link = "https://raw.githubusercontent.com/afshansalahuddin17/Coursera_Capstone/data/df_location_data.csv"
df = pd.read_csv(link,index_col=0)
df.head()

Postcode      Borough                           Neighbourhood
0      M1B  Scarborough                          Rouge ,Malvern
1      M1C  Scarborough  Highland Creek ,Rouge Hill ,Port Union
2      M1E  Scarborough       Guildwood ,Morningside ,West Hill
3      M1G  Scarborough                                  Woburn
4      M1H  Scarborough                               Cedarbrae

#### DF merge to on postcodes

In [8]:
df_new_2 = pd.merge(df, df_new, on='Postcode')
df_new_2.head()

Postcode      Borough                           Neighbourhood   Latitude  \
0      M1B  Scarborough                          Rouge ,Malvern  43.806686   
1      M1C  Scarborough  Highland Creek ,Rouge Hill ,Port Union  43.784535   
2      M1E  Scarborough       Guildwood ,Morningside ,West Hill  43.763573   
3      M1G  Scarborough                                  Woburn  43.770992   
4      M1H  Scarborough                               Cedarbrae  43.773136   

   Longitude  
0 -79.194353  
1 -79.160497  
2 -79.188711  
3 -79.216917  
4 -79.239476

#### csv file output df

In [9]:
df.to_csv(r'df_final_location_data.csv')